contracts/p0/main/Auctioneer.sol

```diff
@@ -99,6 +99,16 @@ contract AuctioneerP0 is
         super.beforeUpdate();
     }
 
+    function owner()
+        public
+        view
+        virtual
+        override(Ownable, SettingsHandlerP0, VaultHandlerP0)
+        returns (address)
+    {
+        return super.owner();
+    }
+
```

Comment:
Oooof - I still don't see why we should thread this through all the contracts -- we _don't_ want to change the idea or behavior of `owner` access between them, right?